### ISSUE TITLE
CIT NBT Support

### ISSUE THREAD

**Issue by JuceDoesThings** [SHsuperCM/CITResewn#44]
Recently I downloaded a pack and it works with Optifine, but not CIT Resewn. It uses NBT data, here's the file:

type=item
items=suspicious_stew
nbt.Effects.0.EffectId=iregex:(23b)

Seems like the whole nbt part of the CIT thing isn't yet finished, just wanted to make sure it was known that it isn't

**Comment by JuceDoesThings** [comment]
[Here's the pack](https://www.planetminecraft.com/texture-pack/visual-stews-optifine/). It works in the latest version with Optifine, but with CITR it does not seem to be reading the nbt properly

**Comment by JuceDoesThings** [comment]
Hm... weird, I might have to update some stuff, gonna see if it works properly then. Also what do you mean by the logs

**Comment by SHsuperCM** [comment]
I mean the game's logs. Make sure the pack is enabled and upload `%appdata%/.minecraft/logs/latest.log`

**Comment by JuceDoesThings** [comment]
Nevermind, I isolated the issue. It seems that, if you have a custom item model AND the CIT NBT data, the custom item model breaks everything. Here, I can quickly slap together a pack of what I mean

**Comment by SHsuperCM** [comment]
I think the issue is a bit more specific than that. Item models and the nbt condition are completely separate and should not affect eachother at all.
CIT is supposed to work natively with models.

**Comment by JuceDoesThings** [comment]
Look, [this is the pack](https://drive.google.com/file/d/1uqKdaybR5UxPwn6g0aDBWffYFoMeZ-g_/view?usp=sharing). The only difference between this and what I sent you is I added the vanilla model for suspicious stew and a texture for it. When I added the item_particle parameter, everything stopped working. I mean they're called bugs for a reason, idk why that happens

**Comment by JuceDoesThings** [comment]
I know this is closed, but I found a workaround to the bug. As long as you specify a new file for every single file mentioned in the custom texture, it fixes the broken textures. Tedious, but possible

**Comment by SHsuperCM** [comment]
Yeah, the fix I pushed in e27a529 just does that automatically(it generates a new file name internally so duplicates wont be overwritten)
